test/config.toml

```diff
@@ -27,6 +32,9 @@ Port = 3498
 NetworkMetricsTick = "43s"
 EthereumMetricsTick = "1m27s"
 
+[maintainer]
```

Comment:
What do you think about renaming it to `[tbtc.maintainer]`? The reasoning is that it's not a generic maintainer but a maintainer specific for tBTC. That would play nicely with the idea of `[bitcoin.electrum]` I think.

Reply:
Done in ed67c93fbe4f57b8aaa434aca80e0100f535f31d.